config/locales/en.yml

```diff
@@ -1457,27 +1457,29 @@ en:
-    project_options_html: "In the bottom row of the project box you can view and add comments <span class='glyphicon glyphicon-comment'></span>, invite collaborators <span class='glyphicon glyphicon-user'></span>, and see the latest activity <span class='glyphicon glyphicon-equalizer'></span> and notifications <span class='glyphicon glyphicon-bell'></span>. You can also edit the project's name and visibility or archive it. You can recall the project from the archive <span class='glyphicon glyphicon-briefcase'></span> at any time. Click on the project name <strong>Demo project - qPCR</strong> to continue."
+    tutorial_welcome_title_html: 'Welcome to <em>sciNote</em>, your electronic lab notebook. sciNote organizes your work in projects. Click <strong>Show projects from</strong> to choose how you want to view your projects. You can also reorder them by clicking the <strong>Sort by</strong> button.'
+    create_project_html: "Click here to create a project and it will appear under the Team you've chosen. Choose public <span class='glyphicon glyphicon-eye-open'></span> if you want everyone from the team to view the project or choose private <span class='glyphicon glyphicon-eye-close'></span> so only the people who you invite to the project can see it."
+    project_options_html: "In the bottom row of the project card you can check the latest activity <span class='glyphicon glyphicon-equalizer'></span>, invite collaborators <span class='glyphicon glyphicon-user'></span>, see the notifications <span class='glyphicon glyphicon-bell'></span> and write comments <span class='glyphicon glyphicon-comment'></span>. You can also edit the project's name and visibility or archive it. You can recall the project from the archive <span class='glyphicon glyphicon-briefcase'></span> in the top right side of the dashboard at any time."
+    create_experiment_html: "Inside of a project you can create experiments. To create one click on this button and edit experiment’s name and description."
+    edit_experiment_html: "By clicking on the down arrow <span class='caret'></span> of an experiment card you can choose between different actions. You can Edit details to the experiment, such as experiment name and description, Clone it inside of a project, where cloned experiment is identical to the original one, Move it from one project to another or Archive it in the archive <span class='glyphicon glyphicon-briefcase'></span>."
     canvas_overview_html: "A task is a basic unit of your experiment. Within a task, you can add protocols, results and add samples. You can connect tasks into workflows to assure traceability of your work. You can grab the task or entire workflow and move it in all directions on the canvas."
```

Comment:
After talking with marketing, could you remove the last sentence: "You can grab the task or entire workflow and move it in all directions on the canvas."